### DISCUSSION TITLE
关注UP主报错

### DISCUSSION THREAD

**Discussion by twtan** [ywmoyue/biliuwp-lite#643] [bug]
### Bug 描述

在视频播放界面点 关注 会报错，但是UP主主页的关注可以正常使用

### 操作步骤

```plain text
1.打开应用
2.播放任意视频
3.点击关注
4.报错
```

### 期望结果

正常在视频播放界面使用关注

### 截图

![image](https://github.com/ywmoyue/biliuwp-lite/assets/79256225/8130b9d9-decf-4f17-a97d-24119ff88633)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

Windows 11 21H2 (Build 22000)

### 设备类型

台式机

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
@ywmoyue 最终的请求中没拿到关注对象的uid. 
https://github.com/ywmoyue/biliuwp-lite/blob/002d31a639477ae1f3c9e3907094aec692a4af4b/src/BiliLite.UWP/ViewModels/User/UserAttentionButtonViewModel.cs#L38C32-L38C33 从这里userid就一直为null了
麻烦看一下

**Reply by ywmoyue** [reply]
好的，非常感谢帮忙定位
